modules/microshift-olm-deploy-ops-spec-ns.adoc

```diff
@@ -0,0 +1,247 @@
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager get pod -l app=catalog-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                READY   STATUS    RESTARTS   AGE
+catalog-operator-7cf77c459d-jbrd7   1/1     Running   0          126m
+----
+
+. Create a namespace by using the following example YAML:
++
+.Example namespace YAML
+[source,YAML]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: olm-microshift
+----
++
+. Apply the namespace configuration using the following command:
++
+----
+$ oc apply -f _<ns.yaml>_ <1>
+----
+<1> Replace _<ns.yaml>_ with the name of your namespace configuration file.
++
+.Example output
+[source,terminal]
+----
+namespace/olm-microshift created
+----
+
+. Create the Operator group YAML by using the following example YAML:
++
+.Example Operator group YAML
+[source,yaml]
+----
+kind: OperatorGroup
+apiVersion: operators.coreos.com/v1
+metadata:
+  name: og
+  namespace: olm-microshift
+spec: <1>
+  targetNamespaces:
+  - olm-microshift
+----
+<1> Omit the `spec.targetNamespaces` field and values for Operators using the global namespace.
+
+. Apply the Operator group configuration by running the following command:
++
+[source,terminal]
+----
+$ sudo oc apply -f _<og.yaml>_ <1>
+----
+<1> Replace _<og.yaml>_ with the name of your operator group configuration file.
++
+.Example output
+[source,terminal]
+----
+operatorgroup.operators.coreos.com/og created
+----
+
+. Create the `CatalogSource` object by using the following example YAML:
++
+.Example catalog source YAML
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: operatorhubio-catalog
+  namespace: openshift-marketplace <1>
+spec:
+  sourceType: grpc
+  image: quay.io/operatorhubio/catalog:latest
+  displayName: Community Operators <2>
+  publisher: OperatorHub.io
+  grpcPodConfig:
+    securityContextConfig: restricted <3>
+  updateStrategy:
+    registryPoll:
+      interval: 60m
+----
+<1> The global namespace. Setting the `metadata.namespace` to `openshift-marketplace` enables the catalog to run in all namespaces. Subscriptions CRs in any namespace can reference catalogs created in the `openshift-marketplace` namespace.
+<2> Community Operators are not installed by default with OLM for {miroshift-short}. Listed here for example only.
+<3> The value of `securityContextConfig` must be set to `restricted` for {microshift-short}.
+
+. Apply the `CatalogSource` configuration by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f _<my-catalog-source.yaml>_ <1>
+----
+<1> Replace _<my-catalog-source.yaml>_ with your catalog source configuration file name.
+
+. To verify that the catalog source is applied, check for `READY` state by using the following command:
++
+[source,terminal]
+----
+$ oc describe catalogsources.operators.coreos.com -n openshift-operators operatorhubio-catalog
```

Comment:
@ShaunaDiaz `openshift-operators` should be `openshift-marketplace`
and also need to update the output of this command

Reply:
Does that include the Service Namespace as well? Note: I spent yesterday working on getting a test env (and TY for the Jenkins). Will try some testing today, see where I get.